docs/guided-tour/import-export/import-data-mappings/README.md

```diff
@@ -0,0 +1,116 @@
+# Import Data Mappings
+
+There are situations where the values in the `DataObjects` are structured differently than the import
+parameters of the blueprint, so that a mapping is necessary. The present example demonstrates such an 
+import data mapping. It is part of the `Installation`.
+
+An import data mapping can for example be useful in the following situation. Suppose there are two `Installations`,
+and the first one exports `DataObjects` which the second one imports. If then the types of the export and import 
+parameters do not match exactly, an import data mapping in the second `Installation` can adapt the data.
+
```

Comment:
What is about advantage setting parameters to fixed values instead of creating e.g. data objects?

Reply:
Done